src/WebJobs.Extensions.DurableTask/Microsoft.Azure.WebJobs.Extensions.DurableTask.xml

```diff
@@ -3451,6 +3451,36 @@
             </summary>
             <param name="context">The orchestration execution context.</param>
         </member>
+        <member name="T:Microsoft.Azure.WebJobs.Extensions.DurableTask.OutOfProcOrchestrationShim.SchemaVersion">
```

Comment:
These are not public APIs, right? It's unfortunate that these comments potentially are getting shipped out with our public API documentation.

Reply:
None of my additions should be public, they're "internal" at most. Would those also affect public API docs? I suspect they shouldn't, but maybe I'm mistaken